maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -911,28 +911,34 @@ class Orchestra(
         config: MaestroConfig?,
         subflowConfig: MaestroConfig?,
     ): Boolean {
-        executeDefineVariablesCommands(commands, config)
-        // filter out DefineVariablesCommand to not execute it twice
-        val filteredCommands = commands.filter { it.asCommand() !is DefineVariablesCommand }
+        // Enter environment scope to isolate environment variables for this subflow
+        jsEngine.enterEnvScope()
```

Comment:
just to be sure - does this work with subflows? In other words, do subflows create their own Orchestra instance at any point - which would accidentally wipe out the env vars? Or  is it one single Orchestra per full execution?